scripts/build/cc/100-gcc.sh

```diff
@@ -403,10 +403,12 @@ do_gcc_core_backend() {
         extra_config+=("--with-system-zlib")
     fi
 
-    # Some versions of gcc have a deffective --enable-multilib.
+    # Some versions of gcc have a defective --enable-multilib.
     # Since that's the default, only pass --disable-multilib.
     if [ "${CT_MULTILIB}" != "y" ]; then
         extra_config+=("--disable-multilib")
+    else
+        extra_config+=("--enable-multiarch")
```

Comment:
Please explain the relations between --enable/disable-multilib and
--enable-disable-multiarch.

Why would we have to --enable-multiarch when CT_MULTILIB = y ?

Do we want to consider that multilib === multiarch from the point of
view of crosstool-NG ? If so, we should state it explicitly, both here
in the code with a big fat comment, *and* in the help text for
CT_MULTILIB in config/target.in

Otherwise, do we need to explicitly --disable-multiarch along with
--disable-multilib (conv. with --enable)> Like so:

    if [ "${CT_MULTILIB}" != "y" ]; then
        extra_config+=("--disable-multilib" "--disable-multiarch")
    else
        extra_config+=("--enable-multilib" "--enable-multiarch")
    fi

Ditto below, of course.